purview/models/__init__.py

```diff
@@ -81,3 +37,71 @@ def replace(self, node, properties=None):   # pragma: no cover
 
 RelationshipManager.disconnect_all = disconnect_all
 RelationshipManager.replace = replace
+
+
+def story_flow(label):
+    """
+    Get the next/previous node in a story flow/pipeline path.
+
+    :param str label: Neo4j node label
+    :return: uid and relationship information in both forward and backward directions
+    :rtype: dict
+    """
+    if label == 'BugzillaBug':
```

Comment:
Optional: I'd prefer if we used `BugzillaBug.__label__` and the same pattern below